lib/pdn/pdn.c

```diff
@@ -317,17 +321,25 @@ int pdn_default_ctx_cb_dereg(pdn_event_handler_t cb)
 	return 0;
 }
 
-static void pdn_ctx_free(struct pdn *pdn)
+static int pdn_ctx_free(struct pdn *pdn)
```

Comment:
This should probably remain `void`.

Reply:
Ok, as long as it is ok not to report an error if the context is already destroyed/does not exist, I am fine with that.